src/master-playlist-controller.js

```diff
@@ -1235,17 +1242,35 @@ export class MasterPlaylistController extends videojs.EventTarget {
   }
 
   /**
-   * Calculates the desired forward buffer length based on current time
+   * Calculates or sets the desired forward buffer length based on current time
    *
-   * @return {Number} Desired forward buffer length in seconds
+   * @param {Number=} Desired forward buffer length in seconds
```

Comment:
According to [this jsdoc page](http://usejsdoc.org/tags-param.html), that's Google Closure-style optional notation, which is used elsewhere in this code base. It's optional because it functions as both the setter and getter.